src/integration/java/io/pinecone/helpers/TestIndexResourcesManager.java

```diff
@@ -37,10 +38,14 @@ public class TestIndexResourcesManager {
     private Pinecone pineconeClient;
     private String podIndexName;
     private IndexModel podIndexModel;
+    private IndexModel serverlessIndexModel;
     private String serverlessIndexName;
     private String collectionName;
     private CollectionModel collectionModel;
-    private final List<String> podIndexVectorIds = Arrays.asList("id1", "id2", "id3");
+    private final List<String> vectorIds = Arrays.asList("id1", "id2", "prefix-id3");
```

Comment:
Since we're adding support for getting a list of things, we should probably test for retrieving multiple things that start with `prefix-`?

Reply:
I'm also thinking it would be helpful--both in terms of readability as well as testing functionality--if we used separate IDs for the set of IDs that are stored in the default namespace vs. those stored in the named namespace. 

That way it'll make test assertions more easy to follow as well as ensure that we don't accidentally verify that the wrong behavior is correct, just b/c the ideas we were checking happened to be the same.

I'm thinking something simple like:

```java
private final List<String> vectorIdsDefaultNs = Arrays.asList("def-id1", "def-id2", "def-prefix-id3", "def-prefix-id4");
private final List<String> vectorIdsCustomNs = Arrays.asList("ns-id1", "ns-id2", "ns-prefix-id3", "ns-prefix-id4");
```

And then activated/used in the appropriate tests

Reply:
Ahh great idea!